data/blogArticles.json

```diff
@@ -0,0 +1,48 @@
+/**** TODO:
+1) I NEED TO COME UP WITH MORE RELEVANT TOPICS, DATA, AND ESP. PROJECTS TO SHOWCASE, AS I HAVE NOT DONE MUCH WITH MAKING THE PORTOLIO HAVE PORTFOLIO-LIKE CONTENT
+2) I NEED TO DECIDE IN WHAT ORDER I WANT TO SHOWCASE THESE TOPIS/ PROJECTS, KEEPING IN MIND THAT FIRST THIS IS DESIGNED FOR MOBILE-VIEW
+3) I NEED TO DECIDE HOW MANY ARTICLES/PROJECTS TO DISPLAY IN SCREEN IN MOBILE-VIEW
+4) I NEED TO DECIDE HOW, OTHER THAN THE WAY WE DID IN LABS, I WANT THE DATA/ ARTICLES, PROJECTS TO INTERACT/ FILTER/SHOW/HIDE, BY ADDING IN TOGGLE, ETC., IN ORDER TO KEEP THE SINGLE-PAGE PORTFOLIO CLEAN, VISUALLY ATTRACTIVE AND EASY TO FOLLOW.****/
+
```

Comment:
I think the suggestions you made were really good. I included many one of the same comments in my own portfolio review. Deciding what relevant data to display is really important. The portfolio is a great place to showcase our web development skills and also our past work but since we don't have a ton it's tricky to know how to display it. I think you're on the right track as far as what macro-changes can be made to the page. As I continue to review your code I will keep in mind that you are planning on making these changes.